src/checkovRunner.ts

```diff
@@ -38,7 +38,7 @@ interface CheckovResponseRaw {
 const skipChecks = ['CKV_AWS_52'];
 
 const dockerMountDir = '/checkovScan';
-const getDockerRunParams = (filePath: string, extensionVersion: string) => ['run', '--tty', '--env', 'BC_SOURCE=vscode', '--env', `BC_SOURCE_VERSION=${extensionVersion}`, '--volume', `"${path.dirname(filePath)}:${dockerMountDir}"`, 'bridgecrew/checkov'];
+const getDockerRunParams = (filePath: string, extensionVersion: string) => ['run', ,'--rm', '--tty', '--env', 'BC_SOURCE=vscode', '--env', `BC_SOURCE_VERSION=${extensionVersion}`, '--volume', `"${path.dirname(filePath)}:${dockerMountDir}"`, 'bridgecrew/checkov'];
```

Comment:
Notice the extra `,` you added 🤓

Reply:
Do not where this come from 😳. Fixed.

Checks passed.